feat(serverless): add more entries in faq/troubleshooting (#3477)
scaleway/docs-content: faq/serverless-containers.mdx

**Monthly Cost: €22.35**

## How to select the right resources (vCPU/RAM) for Serverless Containers ?

> Too low vCPU and RAM values can lead to containers going to error status so it's important to select enough of vCPU and RAM.
> The recommended approach is to set high values and then use metrics [(see documentation)](/serverless/containers/how-to/monitor-container/) to monitor
> the usage of resources and to adjust the value.
> 

Reviewer: ```suggestion
If vCPU and RAM values are too low, it can lead to containers going into error status. It is important to select the correct amount of vCPU and RAM.
The recommended approach is to set high values and then use metrics [(see documentation)](/serverless/containers/how-to/monitor-container/) to monitor
the usage of resources and to adjust the value.
```

## How to reduce cold-start of Serverless Containers ?

> * **Optimise the startup**: Cold-start can be affected by a loading a large number of dependencies and opening lot of resources at startup.

Reviewer: ```suggestion
* **Optimise the startup**: Cold-start can be affected by loading several dependencies and opening many resources at the startup.
```

> * **Optimise the startup**: Cold-start can be affected by a loading a large number of dependencies and opening lot of resources at startup.

Reviewer 2: ```suggestion
* **Optimize the startup**: Cold-start can be affected by a loading a large number of dependencies and opening lot of resources at startup.
```

> Ensure that your code avoid heavy computations or long-running initialization at statup and optimise the number of loaded libraries.

Reviewer: ```suggestion
Ensure that your code avoids heavy computations or long-running initialization at startup and optimizes the number of loaded libraries.
```

> Ensure that your code avoid heavy computations or long-running initialization at statup and optimise the number of loaded libraries.

Reviewer: ```suggestion
Ensure that your code avoids heavy computations or long-running initialization at startup and optimize the number of loaded libraries.
```

> Ensure that your code avoid heavy computations or long-running initialization at statup and optimize the number of loaded libraries.

Reviewer 3: ```suggestion
Ensure that your code avoids heavy computations or long-running initialization at startup and optimize the number of loaded libraries.
```

> * **Keep your container warm**: You can use CRON triggers at certain intervals to keep your container warm or set the min-scale parameter to one when required.

Reviewer: ```suggestion
* **Keep your container warm**: You can use CRON triggers at certain intervals to keep your container warm or set the min-scale parameter to one when required.
```

> * **Increase resources**: Adding more vCPU and RAM can help to reduce significantly the cold-starts of your container.

Reviewer: ```suggestion
* **Increase resources**: Adding more vCPU and RAM can help to significantly reduce the cold-starts of your container.
```

> * **Use sandbox v2**: we recommend to use sandbox v2 (advanced settings) to reduce cold start.

Reviewer: ```suggestion
* **Use Sandbox v2**: we recommend you use Sandbox v2 (advanced settings) to reduce cold start.
```

> * **Use sandbox v2**: we recommend to use sandbox v2 (advanced settings) to reduce cold start.

Reviewer: ```suggestion
* **Use sandbox v2**: We recommend you use sandbox v2 (advanced settings) to reduce cold starts.
```
